### DISCUSSION TITLE
Formatting of angle should probably support different cultures?

### DISCUSSION THREAD

**Discussion by Nickztar** [porrey/Solar-Calculator#14] [General]
For some reason my AngleTest is now failing, is definetly unrelated to this my recent PR but might be worth opening a issue for. It's definetly a culture issue aswell. Probably just in how the formatting works.

This is the result from AngleTest -> IFormattableTest. Same for all cases.
`
String lengths are both 13. Strings differ at index 4.
  Expected: "413°.63911575"
  But was:  "413°,63911575"
  ---------------^
`

Not sure if this is something important so figured I would open a discussion instead of a issue.

**Comment by porrey** [comment]
All of my tests pass so you are probably correct about culture settings.

**Comment by Nickztar** [comment]
Definitely was. Works correctly when adding this to the ShortFormat
`return this.InternalValue.ToString("0°.0000####", System.Globalization.CultureInfo.InvariantCulture);`

**Comment by porrey** [comment]
I updated this in **3.3.0**.
